.vscode/cspell.json

```diff
@@ -790,6 +790,628 @@
         "SEPS"
       ]
     },
+    {
+      "filename": "sdk/ml/azure-ai-ml/NOTICE.txt",
```

Comment:
just add this file to `ignorePaths` at the top of the file instead of listing all of the words here